service/frontend/admin_handler.go

```diff
@@ -145,6 +141,7 @@ type (
 		visibilityMgr                       manager.VisibilityManager
 		Logger                              log.Logger
 		TaskManager                         persistence.TaskManager
+		persistenceExecutionManager         persistence.ExecutionManager
```

Comment:
nit: I think you can change this to just pass in the name, similar to the workflow handler constructor.